POMs/settingsPage.ts

```diff
@@ -8,20 +8,29 @@ export class SettingsPage {
     readonly invoicesLink: Locator;
     readonly logoutButton: Locator;
     readonly confirmButton: Locator;
-    readonly 
+    readonly
 
     constructor(page: Page) {
         this.page = page;
         this.homeLink = page.getByRole('link', { name: 'Home' });
         this.clientsLink = page.getByRole('link', { name: 'Clients' });
         this.offersLink = page.getByRole('link', { name: 'Offers' });
-        this.invoicesLink = page.getByRole('link', { name: 'Invoices'});
+        this.invoicesLink = page.getByRole('link', { name: 'Invoices' });
         this.logoutButton = page.getByText('Log out');
-        this.confirmButton = page.getByRole('button', { name: 'Confirm'})
+        this.confirmButton = page.getByRole('button', { name: 'Confirm' })
     }
 
     async logout() {
         await this.logoutButton.click();
         await this.confirmButton.click();
     }
+
+    async assertLoginPageLink(){
+        await expect(this.page).toHaveURL('https://cobe-accounting.herokuapp.com/');
+    }
+
+    async assertSuccessfulCancellation(){
+        
+    }
```

Comment:
If it is not used, it is better to remove it
But if it will be implemented later, it is better to add comment